ui/src/environments/environment.cromwell.staging.ts

```diff
@@ -4,7 +4,7 @@ export const environment = {
   production: false,
   requiresAuth: false,
   scope: '',
-  additionalColumns: null,
+  additionalColumns: ['cromwell-workflow-name', 'cromwell-workflow-id', 'other-labels'],
```

Comment:
What would go in the `other-labels` column?

Reply:
The `other-labels` for now is just a literal placeholder, we have a ticket in backlog about determining what kind of labels we want to add to workflows, such as Sample id, Workflow type, Workflow version (e.g. v1.2.1), HCA bundle uuid and so on. I'm glad to remove it if that is confusing!